webapp/src/components/card/subscription/index.tsx

```diff
@@ -97,6 +97,14 @@ const SubscriptionCard = ({handleDeleteSubscrption, subscriptionDetails: {channe
                                     {approvalStatusName && <Chip text={`Approval status is: ${approvalStatusName}`}/>}
                                     {approvalTypeName && <Chip text={`Approval type is: ${approvalTypeName}`}/>}
                                     {releaseStatusName && <Chip text={`Release status is: ${releaseStatusName}`}/>}
+                                    {runPipelineName && <Chip text={`Pipeline is: ${runPipelineName}`}/>}
+                                    {runEnvironment && <Chip text={`Environment is: ${runEnvironment}`}/>}
+                                    {runStage && <Chip text={`Stage is: ${runStage}`}/>}
+                                    {runStageId && <Chip text={`Stage is: ${runStageId}`}/>}
```

Comment:
```suggestion
                                    {(runStage || runStageId) && <Chip text={`Stage is: ${runStage || runStageId}`}/>}
```
see if this works, if both cannot exists together